dimod/generators/multi_knapsack.py

```diff
@@ -17,8 +17,62 @@
 from dimod.binary_quadratic_model import BinaryQuadraticModel
 from dimod.constrained import ConstrainedQuadraticModel
 from typing import Tuple
+from dimod.typing import ArrayLike
```

Comment:
Might as well fix the import order while we're at it.

```python
from typing import Tuple

import numpy as np

from dimod.binary_quadratic_model import BinaryQuadraticModel
from dimod.constrained import ConstrainedQuadraticModel
from dimod.typing import ArrayLike
```

Reply:
Updated: https://github.com/dwavesystems/dimod/pull/1390/commits/c75397236d613f7adedf80154df29aaab3a89b89